hypervisor/arch/x86/vmx.c

```diff
@@ -374,22 +408,20 @@ int vmx_write_cr0(struct vcpu *vcpu, uint64_t cr0)
 	uint32_t entry_ctrls;
 	bool paging_enabled = !!(context->cr0 & CR0_PG);
 
-	if ((cr0 & (cr0_always_off_mask | CR0_RESERVED_MASK)) != 0UL) {
-		pr_err("Not allow to set always off / reserved bits for CR0");
+	if (is_cr0_write_valid(vcpu, cr0) == false) {
```

Comment:
is_cr0_write_valid returns Bool. So you don't need " ... == false". I think we can directly use if ( !xxxx). Please check with Junjie

Reply:
OK. Will check with Junjie and update the patch.